nca/FWRules/DotGraph.py

```diff
@@ -159,9 +192,11 @@ def _node_to_str(self, node):
                     table += f'<tr><td>{line}</td></tr>'
             table += '</table>>'
             label = f'label={table}'
-            return f'\t\"{node.name}\" [{label} {self.node_styles[node.node_type]}]\n'
+            node_desc = f'{label} {self.node_styles[node.node_type]} tooltip=\"{self.node_tooltip[node.node_type]}\"'
         else:
-            return f'\t\"{node.name}\" [{self.node_styles[node.node_type]}  xlabel=\"{self.labels_dict[node.label[0]]}\"]\n'
+            node_desc = f'{self.node_styles[node.node_type]}  xlabel=\"{self.labels_dict[node.label[0]]}\" ' \
+                   f'tooltip=\"{self.node_tooltip[node.node_type]}{node.label[0]}\"'
```

Comment:
Why do we add now `{node.label[0]}`?

Reply:
in bi/clique, the node label is the connectivity, we add it also to the tool tip

Reply:
got it